api/migrations/20210525230251-create-user.js

```diff
@@ -11,12 +11,6 @@ module.exports = {
         allowNull: false,
         type: Sequelize.STRING,
       },
-      devBeans: {
```

Comment:
You can't change migrations that have been applied. If you want to remove these columns from the database table, you need to create a new migration file.

Reply:
Looks like this migration hasn't actually be applied to neither the staging nor production database.